### PR TITLE
Adding index path for apf debug endpoints

### LINKED ISSUE #93361: unexpected results from GET /debug
**What happened**:
I tried fetching prefixes of the new debug endpoints, and got a listing of / instead.

```
$ curl http://localhost:8080/debug
{
  "paths": [
    "/apis",
    "/apis/",
    "/apis/apiextensions.k8s.io",
    "/apis/apiextensions.k8s.io/v1",
    "/apis/apiextensions.k8s.io/v1beta1",
    "/healthz",
    "/healthz/etcd",
    "/healthz/log",
    "/healthz/ping",
    "/healthz/poststarthook/crd-informer-synced",
    "/healthz/poststarthook/generic-apiserver-start-informers",
    "/healthz/poststarthook/priority-and-fairness-config-consumer",
    "/healthz/poststarthook/start-apiextensions-controllers",
    "/healthz/poststarthook/start-apiextensions-informers",
    "/livez",
    "/livez/etcd",
    "/livez/log",
    "/livez/ping",
    "/livez/poststarthook/crd-informer-synced",
    "/livez/poststarthook/generic-apiserver-start-informers",
    "/livez/poststarthook/priority-and-fairness-config-consumer",
    "/livez/poststarthook/start-apiextensions-controllers",
    "/livez/poststarthook/start-apiextensions-informers",
    "/metrics",
    "/openapi/v2",
    "/readyz",
    "/readyz/etcd",
    "/readyz/informer-sync",
    "/readyz/log",
    "/readyz/ping",
    "/readyz/poststarthook/crd-informer-synced",
    "/readyz/poststarthook/generic-apiserver-start-informers",
    "/readyz/poststarthook/priority-and-fairness-config-consumer",
    "/readyz/poststarthook/start-apiextensions-controllers",
    "/readyz/poststarthook/start-apiextensions-informers",
    "/readyz/shutdown",
    "/version"
  ]
}$ 
```

```
$ curl http://localhost:8080/debug/api_priority_and_fairness
{
  "paths": [
    "/apis",
    "/apis/",
    "/apis/apiextensions.k8s.io",
    "/apis/apiextensions.k8s.io/v1",
    "/apis/apiextensions.k8s.io/v1beta1",
    "/healthz",
    "/healthz/etcd",
    "/healthz/log",
    "/healthz/ping",
    "/healthz/poststarthook/crd-informer-synced",
    "/healthz/poststarthook/generic-apiserver-start-informers",
    "/healthz/poststarthook/priority-and-fairness-config-consumer",
    "/healthz/poststarthook/start-apiextensions-controllers",
    "/healthz/poststarthook/start-apiextensions-informers",
    "/livez",
    "/livez/etcd",
    "/livez/log",
    "/livez/ping",
    "/livez/poststarthook/crd-informer-synced",
    "/livez/poststarthook/generic-apiserver-start-informers",
    "/livez/poststarthook/priority-and-fairness-config-consumer",
    "/livez/poststarthook/start-apiextensions-controllers",
    "/livez/poststarthook/start-apiextensions-informers",
    "/metrics",
    "/openapi/v2",
    "/readyz",
    "/readyz/etcd",
    "/readyz/informer-sync",
    "/readyz/log",
    "/readyz/ping",
    "/readyz/poststarthook/crd-informer-synced",
    "/readyz/poststarthook/generic-apiserver-start-informers",
    "/readyz/poststarthook/priority-and-fairness-config-consumer",
    "/readyz/poststarthook/start-apiextensions-controllers",
    "/readyz/poststarthook/start-apiextensions-informers",
    "/readyz/shutdown",
    "/version"
  ]
}$ 
```

**What you expected to happen**:
I expected to get a listing of the available ways to extend the URL path I fetched.

**How to reproduce it (as minimally and precisely as possible)**:
As shown above, with API Priority and Fairness enabled.  I test it like this:
```
RUNTIME_CONFIG=flowcontrol.apiserver.k8s.io/v1alpha1=true FEATURE_GATES=APIPriorityAndFairness=true LOG_LEVEL=7 hack/local-up-cluster.sh
```

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
  ```
  mspreitz@mjs-dev-1:~$ kubectl version
  Client Version: version.Info{Major:"", Minor:"", GitVersion:"v0.0.0-master+$Format:%h$",   GitCommit:"$Format:%H$", GitTreeState:"", BuildDate:"1970-01-01T00:00:00Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
  Server Version: version.Info{Major:"1", Minor:"19+", GitVersion:"v1.19.0-alpha.1.3377+33edf0dc89ba6c", GitCommit:"33edf0dc89ba6ccdc480adb477a31e4feed1e883", GitTreeState:"clean", BuildDate:"2020-07-22T21:01:59Z", GoVersion:"go1.14.4", Compiler:"gc", Platform:"linux/amd64"}
  ```
- Cloud provider or hardware configuration: x86 VM with 16 "CPUs" and 32 GiB of memory
- OS (e.g: `cat /etc/os-release`): Ubuntu 18.04.2 LTS
- Kernel (e.g. `uname -a`): Linux mjs-dev-1 4.15.0-109-generic #110-Ubuntu SMP Tue Jun 23 02:39:32 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
- Install tools: hack/local-up-cluster.sh
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by yue9944882**
```release-note
NONE
```

the output will be like: 
```
kimmin-mpb:~ mbp$ curl -L -H 'Authorization: bearer 7b27df02-d21f-4420-844e-76050f1323bc' http://127.0.0.1:50100/debug/api_priority_and_fairness
{
  "paths": [
    "/debug/api_priority_and_fairness/dump_priority_levels",
    "/debug/api_priority_and_fairness/dump_queues",
    "/debug/api_priority_and_fairness/dump_requests"
  ]
}
```

fixes https://github.com/kubernetes/kubernetes/issues/93361

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller_debug.go [hunk 1]
```diff
@@ -37,6 +38,12 @@ const (
 func (cfgCtlr *configController) Install(c *mux.PathRecorderMux) {
 	// TODO(yue9944882): handle "Accept" header properly
 	// debugging dumps a CSV content for three levels of granularity
+	c.UnlistedHandleFunc("/debug/api_priority_and_fairness", func(w http.ResponseWriter, r *http.Request) {
+		http.Redirect(w, r, "/debug/api_priority_and_fairness/", http.StatusFound)
```
Comment (rngy) [inline staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller_debug.go:42]: could this be a status "301 / Permanently Moved" instead ?

**Review by rngy — APPROVED** [review]

**Comment by MikeSpreitzer** [comment]
@MikeSpreitzer

**Comment by dims** [comment]
@yue9944882 is this still needed?

/approve cancel
(to prevent accidental merge)

**Pull request closed without merging**